contracts/FleekERC721.sol

```diff
@@ -41,8 +46,21 @@ contract FleekERC721 is ERC721, FleekAccessControl {
         string gitRepository;
     }
 
-    Counters.Counter private _tokenIds;
+    /**
+     * The stored data for each mirror.
+     * The mirror "score" is used to determine the best mirror to use.
+     * The mirror "score" equals to 0 means that the mirror is not verified.
+     */
+    struct Mirror {
+        uint256 tokenId;
+        uint256 index;
+        uint256 score;
+        address owner;
+    }
+
+    Counters.Counter private _appIds;
     mapping(uint256 => App) private _apps;
+    mapping(string => Mirror) private _mirrors;
```

Comment:
What string value would this be? some unique slug?

Reply:
Maybe hash(token id + index of mirror)? Just something to start with and we can see if review of MVP flows changes anything.

Reply:
The index of the mirror may change when some mirror is removed. Thinking about other choice because this identifier is from where people interact with the mirror, that why it fits well. Imagine the scenario where someone wants to verify a domain, the only way to get to this element directly without having to iterate is when the domain is the key of the mapping.

Reply:
Yeah I need to think about this some more.  We also don't know what future use cases and queries are needed so just thinking what's a good flexible structure to go with.